src/components/Deck.js

```diff
@@ -10,18 +10,37 @@ const buildDeck = () => {
 let deck = buildDeck();
 
 const shuffledDeck = [...deck].sort(() => Math.random() - 0.5);
-const trickDeck = shuffledDeck.slice(0, TOTAL_OF_CARDS);
+const firstColumn = shuffledDeck.slice(0, 7);
+const secondColumn = shuffledDeck.slice(8, 15);
+const thirdColumn = shuffledDeck.slice(15, TOTAL_OF_CARDS+1);
 
 const Deck = () => {
-  const [trickCards, setTrickCards] = useState(trickDeck);
   return(
-    <div className="cards" data-testid ="cards">
-      {trickCards.map(({value, suit}) => {
-        return (
-          <div className ={`card card--${suit}`} key={`${value}-${suit}`}>{value}{suit}</div>
-        )
-      })}
-    </div>
+    <>
+      <div className="cards" data-testid ="first-column">
+        {firstColumn.map(({value, suit}) => {
+          return (
+            <div className ={`card card--${suit}`} key={`${value}-${suit}`}>{value}{suit}</div>
+          )
```

Comment:
Agora que está mais visível o que é a carta no componente (isso fica mais evidente com a repetição dessa linha nos outros casos), o que acha de extrairmos essa parte para um novo componente chamado Card que recebe as props `value` e `suit`? Esse componente não vai ter o `key` adicionado à div interna dele mais, o key permanecerá externo, igual temos aqui. Ficaria algo assim:

```suggestion
          return (
            <Card key={`${value}-${suit}`} value={value} suit={suit} />
          )
```